### ISSUE TITLE
[Client] HTTP 403 on GET:/api/global/self Forbidden (App with Public access after update)

### ISSUE THREAD

**Issue by sergey-dme** [Budibase/budibase#5243]
Hey all, 

I tried to access my app in public mode after updating to 1.0.103, the app page and data are shown but there is a red error message `Forbidden` displayed on the page and also in the console:
`[Client] HTTP 403 on GET: ...api/global/self Forbidden`
API returns: `Unauthorized - no public worker access`

v. 1.0.103

**Comment by mike12345567** [comment]
Hi @sergey-dme - I believe you may be suffering from issue: https://github.com/Budibase/budibase/issues/5158 due to the Envoy -> NGINX migration, can you follow the steps in https://github.com/Budibase/budibase/discussions/5169.

**Comment by sergey-dme** [comment]
Hi @mike12345567 thanks for replying.  
I always respond `Y`to update my `docker-compose.yaml` when asked during `hosting --update`
My current proxy section as follows:
```
  proxy-service:
    restart: unless-stopped
    ports:
      - "${MAIN_PORT}:10000"
    container_name: bbproxy
    image: budibase/proxy
    depends_on:
      - minio-service
      - worker-service
      - app-service
      - couchdb-service
```
 The error message in public mode app I got after the update. But It was working in 1.0.98-99

**Comment by sergey-dme** [comment]
Glad to know that I'm not alone with this

**Comment by sergey-dme** [comment]
Hi @aptkingston is it possible that the issue related with this changes https://github.com/Budibase/budibase/commit/bd93f150b5456459eb5f60d13421bd026517109f ?

**Comment by aptkingston** [comment]
> 

Hey @sergey-dme. Yes that's exactly the change that caused this. I had raised the issue internally that it didn't work for public apps and I thought it had been changed, but it looks like it hasn't. We'll get this fixed soon.

**Comment by FlaminWrap** [comment]
Fixed for me in 1.0.108. Thank you!
